infra/observe/entrypoint.sh

```
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Create experiment directory and ensure it has the correct permissions
```

add `set -e` to fail fast

---

done